src/data/finset/basic.lean

```diff
@@ -1304,6 +1304,12 @@ theorem range_mono : monotone range := λ _ _, range_subset.2
 lemma mem_range_succ_iff {a b : ℕ} : a ∈ finset.range b.succ ↔ a ≤ b :=
 finset.mem_range.trans nat.lt_succ_iff
 
+lemma mem_range_le {n : ℕ} : ∀ x ∈ range n, x ≤ n :=
+  by {  rintros x hx, apply le_of_lt (mem_range .1 hx), }
```

Comment:
```suggestion
lemma mem_range_le {n : ℕ} {x : ℕ} (hx : x ∈ range n) : x ≤ n :=
(mem_range.1 hx).le
```
and similar for the one below. There's no point putting `∀` in the theorem statement unless you plan to use pattern matching - and even then, you should use `∀ {x} (hx : x ∈ range n)` so that the argument explicitness ends up correct.

Reply:
Done!